theme/js/components/utils/read-more.vue

```diff
@@ -72,6 +67,16 @@ export default {
               ease: easing.anticipate,
             }).start(divStyler.set);
           }
+    },
+    updateReadMoreHeight() {
+      let contentHeight = Array.from(this.$refs.container.children)
+      .map(getHeight)
+      .reduce((total, height) => total + height, 0)
+      this.readMoreRequired = contentHeight > getHeight(this.$refs.container);
+      this.containerHeight = DEFAULT_HEIGHT;
```

Comment:
Why do we need to set `containerHeight` again?

Reply:
Explained :)